### ISSUE TITLE
[bug-website module] mangago can check site but not download

### ISSUE THREAD

**Issue by HiP1** [dazedcat19/FMD2#5]
like title says, i cannot download from mangago. but i can browse the mangas and chapters

**Comment by dazedcat19** [comment]
I can't open the website. All links returned `403 forbidden` `we're sorry, the request file are not found. Σ(っ°Д °;)っ`
The website itself seems to be broken. Probably dead.

**Comment by HiP1** [comment]
it's http://mangago.me/
i'm on it right now. 
and their reader seems to be on https://www.lady-first.me/
maybe that's why the downloads fail

**Comment by dazedcat19** [comment]
![img-20200523-123635](https://user-images.githubusercontent.com/59296763/82721673-11a0f000-9cf2-11ea-8a9b-f61b9c955563.jpg)

**Comment by HiP1** [comment]
oh that's weird. have you tried in incognito ?
When i go to there, i get this : 
![Screenshot 2020-05-24 00 36 01](https://user-images.githubusercontent.com/3578646/82741849-f1efe300-9d56-11ea-8311-21c152689197.jpg)

**Comment by dazedcat19** [comment]
Try to ask other if they get the same http response. Can be region related.

**Comment by HiP1** [comment]
what region are you ?
maybe a free VPN could help you ?

**Comment by dazedcat19** [comment]
what ip you got?
>nslookup mangago.me
Server:  dns.google
Address:  8.8.8.8

Non-authoritative answer:
Name:    mangago.me
Addresses:  2606:4700:20::681a:35e
          2606:4700:20::681a:25e
          104.26.3.94
          104.26.2.94

and why you use ABP. that project sold to ads company. ABP, ublock, adblock whatever. all of them owned by ads company. the acceptable ads is a scam itself. those project are dead. there's no more development except some UI and more "acceptable ads". don't be surprised if those software mining your data.
only ublock origin stay true.

**Comment by HiP1** [comment]
seems to be the same as i also use google dns : 
Addresses:  2606:4700:20::681a:25e
2606:4700:20::681a:35e
104.26.2.94
104.26.3.94

i installed abp long ago, and didn't bother changing it.
you use google dns, it's about as fishy as abp in tracking/mining potential :)

**Comment by dazedcat19** [comment]
I actually use bunch of dnscrypt and doh via yogadns server pools. http://yogadns.com/
It automatically intercept every DNS request (even nslookup) and forward it to dnscrypt/doh. So what shown in nslookup server was fake.
And I always block UDP port 53. All of ISPs here intercept DNS request.

It's server issue then. I think I've seen this kind of layout before. It probably a mirror website like mangakakalot. They have a lot of mirror. manganelo. mangastream.today, kissmanga.today. most of those with .today, .to etc. the layout maybe different but when looking at the source and the image urls. it was the same.

**Comment by HiP1** [comment]
maybe your dns setup is what blocks the site content too.

**Comment by dazedcat19** [comment]
if you got the same ip then it was the same. It probably some kind of cdn edge server. that give different responses depend on region. the actual contents probably dead. but it give the cached one to you or the other way around. just try to edit the lua file yourself.

**Comment by HiP1** [comment]
it seems to be a lua error in the page number function :
![image](https://user-images.githubusercontent.com/3578646/82934258-82166e00-9f8b-11ea-9b03-57dc03f4790e.png)

**Comment by HiP1** [comment]
it seems they changed the encryption or added a second layer, and the lua can't match the strings anymore

**Comment by dazedcat19** [comment]
Attach all of the page source that loaded by fmd. You can easily add this line after `HTTP.GET` or something similar. `print(HTTP.Document.ToString())`

**Comment by HiP1** [comment]
[fmd.log](https://github.com/dazedcat19/FMD2/files/4720945/fmd.log)
here is the log from a download attempt that failed.

**Comment by dazedcat19** [comment]
and your `downloads.db`. just strip everything except the failed one.
since I can't open the website I can't make a download.

**Comment by HiP1** [comment]
how do i strip everything else ? i don't have the db engine running. can i do it from FMD ?

**Comment by HiP1** [comment]
here is the file. i had to zip it.
[downloads.zip](https://github.com/dazedcat19/FMD2/files/4730263/downloads.zip)

**Comment by dazedcat19** [comment]
it seems the encryption was changed. it's hard to follow with all the obfuscation. we need someone that really good at javascript.
